ftw/mail/configure.zcml

```diff
@@ -42,6 +42,12 @@
         permission="zope2.View"
         />
 
+    <!-- Mail object creator -->
+    <adapter
+        for="*
+             zope.publisher.interfaces.browser.IBrowserRequest"
```

Comment:
exactly, also see https://github.com/4teamwork/opengever.core/pull/2930 for an example. btw, for our current use-case context would not be required since this is more a global thing, but i figured id does not hurt and adds more flexibility.